exercises/accumulate/example.go

```diff
@@ -1,5 +1,8 @@
 package accumulate
 
+// TestVersion constant declaration - part of the boilerplate for exercism.
```

Comment:
I think we can leave this comment out, it'll be covered in the README and is only necessary in stub files for exercises that have them. This is another thing that is not consistent across the exercises, I'll add it to #470